dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -527,7 +528,7 @@
     <!-- The Name of the Local Assembly in Managed and Native -->
     <PropertyGroup>
       <LocalAssembly Condition="'$(LocalAssembly)' == '' and Exists(@(XamlIntermediateAssembly))">
-        @(XamlIntermediateAssembly->'%(Identity)')
+        @(XamlIntermediateAssembly-&gt;'%(Identity)')
```

Comment:
>-&gt; [](start = 34, length = 5)

expect ->

Reply:
The CreateNewProject script is the one doing this, because it writes the project file back in proper xml. It is harmless.